tools/pipeline-witness/infrastructure/kusto/functions/users/kojamroz/SpecsPipelinesBuildsJobLogs.kql

```diff
@@ -1,57 +0,0 @@
-.create-or-alter function with (folder='users/kojamroz', docstring='Build logs for specific ADO job in the specs pipelines builds. See source for details.') SpecsPipelinesBuildsJobLogs(jobName:string, start:datetime, end:datetime, buildId:string="*")
```

Comment:
I would love if we could avoid deleting stuff in my `users/kojamroz` folder. I added these with the intent of being backup sources. So not sure why they have to be deleted?

Reply:
The deployment scripts weren't run in previous PRs. They gather all of the kusto files in the infrastructure folder and deploy them as part of the bicep deployment to either test, staging or production.  So, committing kusto scripts to the kusto folder should have the intent of putting objects in the production database.

Reply:
For this file, [SpecsPipelinesBuildsJobLogs.kql](https://github.com/Azure/azure-sdk-tools/pull/8147/files/204c099e9c43760a612ff0b61f044794670f2e19#diff-7717972af68bf5e6719fcbe2fcabb83d7afee391d2331549e06050c6236068c8), the cross cluster table access breaks deployment to test, where a fresh cluster is provisioned in Playground without access to the prod cluster.

Reply:
@hallipr how about we move these sources to some location that won't be picked up by any automation? Can you do that in this PR instead of deleting these files?

Reply:
I'm not sure they belong in pipeline-witness project if they're not part of the product.  Perhaps just add them to the wiki

Reply:
If a process depends on the functions in the prod database, we should fix them so they're deployable to test and staging.  Otherwise, we should add them to the [Kusto Query Catalog](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/625/Kusto-Query-Catalog)

Reply:
OK I'll copy them over to the wiki. Feel free to keep them deleted in this PR. Thanks for letting me know.

Reply:
@hallipr you mention this is broken due to "cross-cluster". But in fact this function is using only one cluster. I spelled it out to make it explicit. If we would remove the `cluster` prefix would it start to work? If so, how can I ensure it is uploaded both to PROD and PPE/TEST clusters? Can you point me to relevant pipeline so I can experiment with fixing the query?

In any case, feel free to delete the code from this PR for now. If I manage to fix it I will create another PR.

Reply:
@konrad-jamrozik, the pipeline is here:
https://dev.azure.com/azure-sdk/internal/_build?definitionId=1716&_a=summary

I recently updated it to do full bicep deployment in ci, so we should see deployment breaks as soon as the code hits main.  You can also run the pipeline against your branch or PR and only deploy to Staging to test the kql bundle / bicep deployment

Reply:
You can test the deployment by running `/tools/pipeline-witness/infrastructure/deploy.ps1 -target test`.  This will deploy to Playground